docs/vectorizer/api-reference.md

```diff
@@ -1384,23 +1516,30 @@ You use `ai.disable_vectorizer_schedule` to:
 
 #### Example usage
 
-To stop the automatic scheduling for the vectorizer with ID 1.
+To stop the automatic scheduling for a vectorizer:
 
 ```sql
+-- Using ID
 SELECT ai.disable_vectorizer_schedule(1);
+
+-- Using name (recommended)
+SELECT ai.disable_vectorizer_schedule('public_blog_embeddings');
 ```
 
 #### Parameters
 
-`ai.disable_vectorizer_schedule` takes the following parameters:
+`ai.disable_vectorizer_schedule` takes one of the following parameters:
```

Comment:
this is misleading. There are now two functions one that takes an id and one that takes a name. Note one that takes an id or a name.

Reply:
okay I felt like two tables is more confusing than this, but changed it everywhere, let me know what you think.